docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md.orig

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
should this file be deleted?

Reply:
Yep, sorry, a leftover from solving the merge conflict!